testnets/qubeticstestnet/chain.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "qubeticstestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Qubetics Testnet",
+  "chain_type": "cosmos",
+  "chain_id": "qubetics_9029-1",
+  "bech32_prefix": "qubetics",
+  "daemon_name": "qubeticsd",
+  "node_home": "$HOME/.tmp-qubeticsd",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "tics",
+        "low_gas_price": 20000000000,
+        "average_gas_price": 25000000000,
+        "high_gas_price": 40000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "tics"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/Qubetics/qubetics-chain",
+    "recommended_version": "v0.0.1",
+    "compatible_versions": [
+      "v0.0.1"
+    ],
+    "consensus": {
+      "type": "cometbft",
```

Comment:
this says 'cometbft' here, but tendermint in the versions file. Please make sure they agree.